pages/about.js

```diff
@@ -241,6 +245,16 @@ const TutorialPage = () => {
       <div className={classes.root}>
         <TutorialHeader />
         <section className={classes.info}>
+          <p>
+            <iframe
+              className={classes.video}
+              src="https://www.youtube.com/embed/WfdfB7GyqMY"
+              title={t`Cofacts fact-checking chatbot, combating misinformation by yourself!`}
```

Comment:
Here I use the Youtube video title as the iframe title. Are there any idea or it's ok to use it?

Reply:
It is great! Thank you for picking the title!